addons/hr/static/tests/disable_patch.js

```diff
@@ -0,0 +1,6 @@
+/** @odoo-module */
+
+import { unpatch } from "@web/core/utils/patch";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+
+unpatch(AvatarCardPopover.prototype, "hr");
```

Comment:
how does it work? onClickViewEmployee is not defined with that but the xpath is still applied

Reply:
ok for now but it is not ideal. We should find a solution at some point